docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -8,231 +8,175 @@ This document provides Cobalt users instructions for initializing and integratin
 
 ### Prerequisites
 
-  * An Azure Subscription
-  * Azure Devops Organization
-  * Permissions to your Organization's Azure Devops account
-  * *Global administrator role* permissions in your Organization's Azure Active Directory tenant to setup service principals
+  1. An Azure Subscription
+  2. Azure Devops Organization
+  3. Permissions to your Organization's Azure Devops account
+  4. *Global administrator role* permissions in your Organization's Azure Active Directory tenant to setup service principals
     * If this is not allowed by your organization, step two and the Service Connection creation in step three will need to be completed by someone within your organization with this permission.
 
-> If using the CLI commands please set the following variables:
-
-| Variable | Sample Value | Description |
-|----------|--------------|-------------|
-| `TEMPLATE_DEVOPS_PROJECT_NAME` | `My Application` | The name of the project representing your Cobalt template application that serves as your organization's advocated pattern for a specific template. |
-| `TEMPLATE_DEVOPS_INFRA_REPO_NAME` | `az-hello-world` | The name of the repo that will be created in the application Azure DevOps project to host the Cobalt template. |
-| `TEMPLATE_DEVOPS_INFRA_YML_PATH` | `devops/providers/azure-devops/templates/azure-pipelines.yml` | The path relative to the `TEMPLATE_DEVOPS_INFRA_REPO_NAME` root that contains the Cobalt template pipeline to be created for testing and provisioning resources. |
-| `DEFAULT_ORGANIZATION` | `https://dev.azure.com/MyOrganization/` | The full URL path of the organization in which your `TEMPLATE_DEVOPS_PROJECT_NAME` resides or will be created. |
-| `COBALT_SOURCE_URL` | `https://github.com/microsoft/cobalt.git` | The Git clone URL for Cobalt (containing all templates including the one to be targeted by this template repository) from which this Cobalt template repository will be sourced. |
-
-Update these values for your environment and application based on the guidance in the table above.
-```bash
-export TEMPLATE_DEVOPS_PROJECT_NAME=""
-export TEMPLATE_DEVOPS_INFRA_REPO_NAME=""
-export TEMPLATE_DEVOPS_INFRA_YML_PATH=""
-export DEFAULT_ORGANIZATION=""
-export COBALT_SOURCE_URL=""
-export SUBSCRIPTION_ID=""
-export SUBSCRIPTION_NAME=""
-export TENANT_ID=""
-export SERVICE_PRIN_ID=""
-export SERVICE_CONN_NAME=""
-export AGENT_POOL_NAME=""
-```
-
-The following values are used like constants and should not need to change (unless the build pipeline definition is modified).
-
```

Comment:
The cli commands are very helpful for those who don't want to use the UI and they've all been removed from this file. I vote to add those back. Can those be added back?